client/src/components/Donation/donate-form.tsx

```diff
@@ -332,14 +332,25 @@ class DonateForm extends Component<DonateFormProps, DonateFormComponentState> {
     )}:`;
     const showMinimalPayments = isSignedIn && (isMinimalForm || !isDonating);
 
+    const isAsVariant = !isAVariant;
+
     return (
       <>
         <b className={isMinimalForm ? 'donation-label-modal' : ''}>
-          {isAVariant === false && <SecurityLockIcon />}
           {this.getDonationButtonLabel()}:
         </b>
         <Spacer />
-        <div className={'donate-btn-group'}>
+        <fieldset
+          className={`donate-btn-group ${
+            isAsVariant === false ? 'test-btn-group' : ''
```

Comment:
```suggestion
            isAVariant ? 'test-btn-group' : ''
```

Reply:
I readjusted the logic so this change is not necessary.